spec/javascripts/admin/modules/unpublish-display-conditions.spec.js

```diff
@@ -95,4 +105,13 @@ describe('GOVUK.Modules.UnpublishDisplayConditions', function () {
     expect(body.querySelector('.js-unpublish-withdraw-form__consolidated').style.display).toEqual('block')
     expect(body.querySelector('.js-unpublish-withdraw-form__withdrawal').style.display).not.toEqual('block')
   })
+
+  it('should not show Public explanation section when Redirect to URL automatically is checked', function () {
+    var checkbox = body.querySelector('.govuk-checkboxes__item input[name="unpublishing[redirect]"]')
+    console.log(checkbox)
+    checkbox.checked = true
+    checkbox.dispatchEvent(new Event('change'))
+
+    expect(body.querySelector('.js-unpublish-withdraw-form__published-in-error div.gem-c-textarea').style.display).not.toEqual('block')
+  })
```

Comment:
Probably want another test to see if it shows when the checkbox is already checked.